frontend/webadmin/modules/gwt-common/src/main/resources/org/ovirt/engine/ui/common/CommonApplicationConstants.properties

```diff
@@ -636,6 +636,7 @@ registerDisksLabel=Register Disks
 remoteDesktop=Remote Desktop
 removeConfirmationPopupMessage=Are you sure you want to remove the following items?
 removeDisk=Remove
+removeDirectSystemPermissions=Remove Direct System Permissions
```

Comment:
1. Moved, thanks. Initially I thought about it. But as we can see I don't know the alphabet well =)
2. Added, i think you right
3. I thought about this for several hours. The problem is in content for this list. There we have "all" roles. From groups, from "anyone" object and direct for user. I think that I can confuse user if button will have label "Delete All", because the button will delete only "direct permissions".
I tested with narrow window, and buttons summary not so wide. When I did smallest window width in chrome, buttons still in single row and with full labels.